app/serializers/api/order_serializer.rb

```diff
@@ -16,7 +16,7 @@ def outstanding_balance
     end
 
     def payments
-      object.payments.joins(:payment_method).completed
+      object.payments.joins(:payment_method).valid
```

Comment:
There's a failing test in the order serializer spec, I'm assuming it's due to this line.

Reply:
9aecefb05d24250e0168b99cf6ea53e54ebe393f - I looked around for other places that changing this might impact but I didn't see any that we're explicitly expecting pending payments to **not** be returned.